### PR TITLE
Added UG document for AIAssistView GettingStrated

### PR DESCRIPTION
**Pull request by RavikumarAkuthotaSf4250**
Added UG document for AIAssistView GettingStrated

### CODE REVIEW

File: MAUI/AIAssistView/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+          });
+      }
+   }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Binding AssistItems to SfAIAssistView
+
+To bind the assistItems to `SfAIAssistView`, configure the `AssistItems` property accordingly. You have the option to bind the assistItems collection of the `SfAIAssistView` either through XAML or programmatically in code.
```
Comment (PiruthivirajM) [inline MAUI/AIAssistView/getting-started.md:159]: To bind the assist items

File: MAUI/AIAssistView/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+          {
+              Text = "Sure! MAUI stands for .NET Multi-platform App UI. It’s a framework that allows you to create cross-platform applications using a single codebase.This powerful framework is an evolution of Xamarin.Forms and is designed to streamline the development process by allowing you to write code once and deploy it across multiple platforms.",
+          });
+      }
+   }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Binding AssistItems to SfAIAssistView
```
Comment (PiruthivirajM) [inline MAUI/AIAssistView/getting-started.md:157]: Binding AssistItems to AI AssistView

File: MAUI/AIAssistView/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+platform: MAUI
+control: SfAIAssistView
+documentation: ug
+---
+
+# Getting Started with .NET MAUI AI Assist View (SfAIAssistView)
+
+This section provides a quick overview for getting started with the .NET MAUI AI Assist View (SfAIAssistView). Walk-through the entire process of creating real world of this control.
+
+## Creating an application using the .NET MAUI SfAIAssistView
```
Comment (PiruthivirajM) [inline MAUI/AIAssistView/getting-started.md:14]: here say control name as "AI AssistView",
follow the same for all headings.

Comment (RavikumarAkuthotaSf4250) [reply MAUI/AIAssistView/getting-started.md:14]: ok sir

File: MAUI/AIAssistView/getting-started.md (hunk 1 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/AIAssistView/getting-started.md]: Made the changes

File: MAUI/AIAssistView/getting-started.md (hunk 2 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/AIAssistView/getting-started.md]: Made the requested changes

**Review by PiruthivirajM — APPROVED** [review]
This is fine

**Pull request merged**